examples/gst.js

```diff
@@ -0,0 +1,35 @@
+const GiJS = require('../lib/')
```

Comment:
Keep it consistent with the rest, use `gi` as the variable name. If you have a better name, still follow JS conventions (lowerCamelCase for variables).

Reply:
Fixed